affirm/src/main/java/com/affirm/android/model/CardDetails.java

```diff
@@ -17,18 +19,22 @@
         return new AutoValue_CardDetails.GsonTypeAdapter(gson);
     }
 
+    @Nullable
```

Comment:
Are these fields that should be nullable? That seems like it would be a problem

Reply:
replied via DM